pkgs/applications/networking/n8n/default.nix

```diff
@@ -17,6 +17,12 @@ nodePackages.n8n.override {
   meta = with lib; {
     description = "Free and open fair-code licensed node based Workflow Automation Tool";
     maintainers = with maintainers; [ freezeboy k900 ];
-    license = licenses.asl20;
+    license = {
+      fullName = "Sustainable Use License";
+      url = "https://github.com/n8n-io/n8n/blob/fa7b12c3a3e81b86c76b6490b0d6e4f81115ded8/LICENSE.md";
```

Comment:
```suggestion
      url = "https://github.com/n8n-io/n8n/blob/master/LICENSE.md";
```

Reply:
Maybe link to the 0.169.0 tag then? It's a custom-written license, so I intentionally linked a commit in case they decide to alter the wording.